.devops/nix/package-gguf-py.nix

```diff
@@ -0,0 +1,20 @@
+{
+  lib,
+  llamaVersion,
+  python3,
```

Comment:
`buildPythonPackage`, `poetry-core`, `numpy` as arguments

Reply:
The evaluating scope should set them explicitly then?

Reply:
Not sure if there's a cleaner way: [4957125](https://github.com/ggerganov/llama.cpp/pull/5664/commits/4957125daf29cae8e1ac3d1a81601ca765e080d2)

Reply:
No, the scope inherits from `python3Packages`, which in turn inherits from `pkgs`

Reply:
Ah, sorry, mistake on my part. This one only inherits from `pkgs`, we need to handle python separately

Reply:
If by "good to go" you mean we need to move the python scripts out of the main derivation so that it doesn't depend on the choice of python version 😅